src/registration.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { useForm } from "react-hook-form"
+import { useNavigate } from "react-router-dom";
+
+export const Registration = ()=> {
+  const { register, handleSubmit } = useForm()
+  const navigate = useNavigate()
+  
+  const onSubmit = async (dataReg) => {
+    let encodedObject = encodeURIComponent(JSON.stringify(dataReg));
+    const res = await fetch(`http://localhost:8008/Registration=${encodedObject}`)
+    const resJson = await res.json()
+    if (typeof resJson === "object" ){
+      alert("kongratulejszyn")
+    }
+    else{alert("kajak od tyłu to wciąż kajak, a ty od tyłu jestes jebany, bo ktoś juz ci mejla zajumał")}
+    
+  }
+  return (
+    <div className="registration">
+    <form onSubmit={handleSubmit(onSubmit)}>
+      <label>
```

Comment:
Same as in https://github.com/KAR-HUD-SOFTWARE-DEVELOPMENT/landing-page/pull/1/files#r1591481826